index.html

```diff
@@ -1,9 +1,11 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<!DOCTYPE HTML>
 <html>
 	<head>
 		<title>Alex Nisnevich :: Portfolio</title>
-		<base target="_blank">
```

Comment:
Why did you take away the `<base target=_blank>`? I want all links to open in another tab by default, and I can't think of a better way to do that.